.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - checkout
       - run: luarocks make *.rockspec
       - run: /usr/local/openresty/luajit/bin/apicast-cli busted
-      - run: prove
+      - run: TEST_NGINX_BINARY=openresty-debug prove
```

Comment:
You'll probably have to roll this back and use debug only locally. Can't get working test on Travis.